broker/broker-app/src/main/resources/features.xml

```diff
@@ -0,0 +1,157 @@
+        <bundle>mvn:org.apache.activemq/artemis-amqp-protocol/${artemis.version}</bundle>
+    </feature>
+
+    <feature name="artemis-stomp" install="manual" version="${artemis.version}"
+             description="ActiveMQ Artemis Stomp protocol libraries">
+        <feature prerequisite="true">artemis-core</feature>
+        <bundle>mvn:org.apache.activemq/artemis-stomp-protocol/${artemis.version}</bundle>
+    </feature>
+
+    <feature name="artemis-mqtt" install="manual" version="${artemis.version}"
+             description="ActiveMQ Artemis MQTT protocol libraries">
+        <feature prerequisite="true">artemis-core</feature>
+        <bundle>mvn:io.netty/netty-codec-mqtt/${netty.version}</bundle>
+        <bundle>mvn:org.apache.activemq/artemis-mqtt-protocol/${artemis.version}</bundle>
+    </feature>
+
+    <feature name="artemis-hornetq" install="manual" version="${artemis.version}"
+             description="ActiveMQ Artemis HornetQ protocol libraries">
+        <feature prerequisite="true">artemis-core</feature>
+        <bundle>mvn:org.apache.activemq/artemis-hqclient-protocol/${artemis.version}</bundle>
+        <bundle>mvn:org.apache.activemq/artemis-hornetq-protocol/${artemis.version}</bundle>
+    </feature>
+
+    <feature name="artemis-openwire" install="manual" version="${artemis.version}"
+             description="ActiveMQ Artemis HornetQ protocol libraries">
+        <feature prerequisite="true">artemis-core</feature>
+        <bundle>mvn:org.fusesource.hawtbuf/hawtbuf/1.11</bundle>
+        <bundle>mvn:org.apache.geronimo.specs/geronimo-j2ee-management_1.1_spec/1.0.1</bundle>
+        <bundle>mvn:org.apache.activemq/artemis-openwire-protocol/${artemis.version}</bundle>
+    </feature>
+
+    <feature name="broker-adapter" install="manual" version="${project.version}"
+             description="Fragment used to integrate Artemis with DDF">
+        <bundle>mvn:org.codice.ddf.broker/broker-adapter/${project.version}</bundle>
+    </feature>
+
+    <feature name="broker-connection-factory" install="manual" version="${project.version}"
+             description="Common jms connection factory configuration">
+        <feature prerequisite="true">artemis</feature>
+        <bundle>mvn:org.codice.ddf.broker/broker-connection-factory/${project.version}
+        </bundle>
+    </feature>
+
+    <feature name="broker-topic-logger" install="auto" version="${project.version}"
+             description="Common topic sample and throughput logger">
+        <feature prerequisite="true">broker-connection-factory</feature>
+        <feature prerequisite="true">camel-sjms</feature>
+        <bundle>mvn:org.codice.ddf.broker/broker-topic-logger/${project.version}</bundle>
+    </feature>
+
+    <feature name="broker-app" install="auto" version="${project.version}"
+             description="The Broker Application provides broker services for reliable message delivery in the DDF.">
```

Comment:
So currently the title of the broker app that's displayed in the Admin UI is "Broker App", we should rename it to just "Broker". Add "::Broker" at the end of the description to change the title.